### PR TITLE
FUSETOOLS2-39: Create UI Tests for find references for direct and dir…

### PR DESCRIPTION
**Pull request by pospisilf**
…ect VM components

# Pull Request informations

## Rebase & Merge default requirements

1. Green build for main branch
2. Wait 24 hours after PR creation
3. Green job for PR
4. Approved PR

## PR labels default process

- READY_FOR_REVIEW  &rightarrow;  REVIEW_DONE  &rightarrow;  READY_FOR_MERGE

## Tests

- [ ] Are there **Unit tests**?
- [ ] Are there **Integration tests**?
- [ ] Do we need a new **UI test**?

## PR workflow progress

1. [ ] Tagged with relevant **PR labels**
2. [ ] Green **job for PR**
3. [ ] PR was created more than **24 hours ago** or **All committers approved** it
4. [ ] Green **main** branch build

### CODE REVIEW

File: src/ui-test/tests/dsls.test.ts [hunk 1]
```diff
@@ -461,3 +498,44 @@ async function autocompletionForReferenceIDofDirectVMComponent(uriLine: number,
...
+	// wait until 'References' refreshed
+	await driver.wait(async function () {
+		return (await isReferencesAvailable());
+	}, 15000);
+
+	const section = await new SideBarView().getContent().getSection('References') as DefaultTreeSection;
+	const visibleItems = await section.getVisibleItems();
+	assert.equal(await visibleItems.at(0).getLabel(), REFERENCES_FILE_2);
+	assert.equal(await visibleItems.at(1).getLabel(), expectedReference);
+}
```
Comment (pospisilf) [inline src/ui-test/tests/dsls.test.ts:541]: no empty line

File: src/ui-test/utils/testUtils.ts [hunk 2]
```diff
@@ -398,3 +406,49 @@ export async function createNewFile(driver: WebDriver, name: string): Promise<vo
...
+		await section.click();
+		await new Workbench().executeCommand('references-view.clear'); // clear current if available - must be done
+		await new Workbench().executeCommand('references-view.clearHistory'); // clear history
+	}
+	catch (error) {
+		console.log('error' + error); // if 'References' Side Bar is not opened
+	}
+}
+
+
```
Comment (pospisilf) [inline src/ui-test/utils/testUtils.ts:454]: redundant empty line

File: src/ui-test/utils/testUtils.ts [hunk 3]
```diff
@@ -398,3 +406,49 @@ export async function createNewFile(driver: WebDriver, name: string): Promise<vo
...
+ */
+export async function clearReferences(): Promise<void> {
+	try {
+		const section = await new SideBarView().getContent().getSection('References') as DefaultTreeSection;
+		await section.click();
+		await new Workbench().executeCommand('references-view.clear'); // clear current if available - must be done
+		await new Workbench().executeCommand('references-view.clearHistory'); // clear history
+	}
+	catch (error) {
+		console.log('error' + error); // if 'References' Side Bar is not opened
```
Comment (pospisilf) [inline src/ui-test/utils/testUtils.ts:450]: maybe not necessary?

**Review by apupier — APPROVED** [review]

**Review by djelinek — APPROVED** [review]
fixed some typos and the rest of changes looks good to me 🙂

**Pull request merged**